homeassistant/components/sonos/diagnostics.py

```diff
@@ -130,11 +130,11 @@ def get_contents(
         value = getattr(speaker, attrib)
         payload[attrib] = get_contents(value)
 
-    payload["enabled_entities"] = {
+    payload["enabled_entities"] = sorted(
         entity_id
         for entity_id, s in hass.data[DATA_SONOS].entity_id_mappings.items()
         if s is speaker
-    }
+    )
```

Comment:
Sorting the entities to make the list easier to read in diagnostics and also to allow the snapshot assertion to work reliably.